vcpkg.json

```diff
@@ -31,20 +31,25 @@
             "wayland"
           ],
           "platform": "linux"
-        }
+        },
+        "curl",
+        "libarchive",
```

Comment:
libarchive includes crypto support by default, but it seems like we don't really need to support encrypted archives right?

Reply:
not really

Reply:
then let's not enable crypto :) (and anything other than zip support is probably unnecessary too?)

```json
{
    "name": "libarchive",
    "default-features": false
}
```

Reply:
ye, if we want to support tar and other stuff latter on, we can still re-enable it.
Is this what I should replace the current entry with?

Reply:
yup

Reply:
done